serverless-cljs-plugin/serverless_lumo/build.cljs

```diff
@@ -34,11 +37,12 @@
        (.on archiver "error" reject-fn)
        (.pipe archiver output-stream)
        (doseq [d (:dirs zip-opts)]
-         (.directory archiver d))
+         (.directory archiver d #js {:mode 755}))
        (doseq [f (:files zip-opts)]
          (if (string? f)
-           (.file archiver f)
-           (.apply (.-file archiver) archiver (clj->js (seq f)))))
+           (.file archiver f #js {:mode 644})
+           (.apply (.-file archiver) archiver
```

Comment:
This line is quite hard to follow - if it's correct, can you bind the `update` expression in a `let` and use it by name inside .apply?

Reply:
I sure can do that.